Sources/JWTKit/JWK.swift

```diff
@@ -5,6 +5,36 @@
 public struct JWKS: Decodable {
     /// All JSON Web Keys
     public var keys: [JWK]
+
+    /// Retrieves the desired key from the JSON Web Key Set
+    /// - Parameters:
+    ///   - keyIdentifier: The `kid` value to lookup.
+    ///   - keyType: The `kty` value.
+    public func find(keyIdentifier: String, keyType: JWK.KeyType) -> JWK? {
```

Comment:
Can this  be `find(identifier:type)`? Having `key` here seems redundant to me.

Reply:
Sure.  I just used those names because that's what the property was called.